lib/ansible/modules/network/radware/vdirect_file.py

```diff
@@ -0,0 +1,243 @@
+#!/usr/bin/python
+#  -*- coding: utf-8 -*-
+#
+# Copyright 2017 Radware LTD.
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.0'}
+
+DOCUMENTATION = '''
+module: vdirect_file
+author: Evgeny Fedoruk (@evgenyfedoruk)
+short_description: Uploads a new or updates an existing runnable file into Radware vDirect server
+description:
+    - Uploads a new or updates an existing configuration template or workflow template into the Radware vDirect server.
+      All parameters are not mandatory required since they may be set as environment variables.
+notes:
```

Comment:
There is a `requirements:`  than should be used for this, see other modules for examples